components/Footer.js

```diff
@@ -13,7 +13,7 @@ export default function Footer (props) {
           <ul>
             <li>
               <NavLink
-                // FIXME: Update content at link
+                // FIXME: Update content at link OR change to Quick Start.
```

Comment:
What is Quick Start?

Reply:
A non-existent component that might be useful to have for API users. I can update comment.